--- origsrc/PAR-Packer-1.014/myldr/Makefile.PL	2012-12-21 08:31:47.000000000 -0600
+++ src/PAR-Packer-1.014/myldr/Makefile.PL	2013-06-05 14:49:39.172467300 -0500
@@ -115,11 +115,11 @@ if( $cc =~ m/^cl\b/i ) {
     $lddebug = ($debug or $^O eq 'darwin') ? '' : '-s ';
     $warn = $debug ? '-Wall -Wno-comments ' : '';
     if ( $^O =~ /^(?:MSWin|cygwin)/ ) {
-        my $target = $Config{archname} =~ /x64/ ? 'pe-x86-64' : 'pe-i386';
+        my $target = $Config{myuname} =~ /x86_64/ ? 'pe-x86-64' : 'pe-i386';
         $res = 'ppresource.coff';
-        $pre_res = qq(winres\\pp.rc);
+        $pre_res = qq(winres/pp.rc);
         $rt_cmd = '-$(NOOP)';
-        $res_cmd = qq(windres -i winres\\pp.rc -o ppresource.coff --input-format=rc --output-format=coff --target=$target);
+        $res_cmd = qq(windres -i winres/pp.rc -o ppresource.coff --input-format=rc --output-format=coff --target=$target);
         $res_section = $res;
     }
     $mt_cmd = '-$(NOOP)';
